class/Files/Classes/ClassSpecialFiles.php

```diff
@@ -267,7 +267,7 @@ public function renderConstants()
         return $this->renderFile();
     }
 
-    public function renderConstantsTestInterface()
+    public function renderConstantsInterface_Version_Zyspec()
```

Comment:
PSR-1: Method names MUST be declared in camelCase
https://www.php-fig.org/psr/psr-1/

Reply:
Hi Michael
the function recommended by zyspec is not working and causes an error. I wrote him a mail but got no answer till now :(

Reply:
That's fine, but the function should be then as:
renderConstantsInterfaceVersionZyspec
without the underscores